macros/create_proxy_views.sql

```diff
@@ -0,0 +1,49 @@
+{% macro create_proxy_views(models) %}
+
+{% set prod_schema = "analytics" %} -- This is the schema name of our production schema
+{% if prod_schema == target.schema %}
+    {% do log("This macro shouldn't be run on the production target. Exiting without actions.", info=True) %}
+{% else %}
+    {% for model in models %}
+        {% set relation_identifier = model["alias"] %}
+        {% set relation_sub_schema = model["config"]["schema"] %}
```

Comment:
🥖 -crumb: For our redshift project this is not so relevant, but in case people [use custom schemas](https://docs.getdbt.com/docs/building-a-dbt-project/building-models/using-custom-schemas), we would like to make sure we build the objects accordingly.